src/config.rs

```diff
@@ -52,8 +52,14 @@ pub struct Config {
 	#[serde(rename = "LIBREDDIT_DEFAULT_HIDE_AWARDS")]
 	pub(crate) default_hide_awards: Option<String>,
 
+	#[serde(rename = "LIBREDDIT_DEFAULT_SUBSCRIPTIONS")]
+	pub(crate) default_subscriptions: Option<String>,
+
 	#[serde(rename = "LIBREDDIT_BANNER")]
 	pub(crate) banner: Option<String>,
+
+	#[serde(rename = "LIBREDDIT_PUSHSHIFT_FRONTEND")]
+	pub(crate) pushshift: String,
```

Comment:
@sigaloid Should this be the cause for the failed test? Looks like as if this should be `Option<String>`?

Reply:
It needs to be String, as there must be a frontend specified. In any case I don't think that could really make it fail, because it's not affecting the existing test cases... No new tests were added. :/

Reply:
Ok, i see. I was just trying to have a look at the failed test case but i have to admit that i never worked with Rust before. Just saw that the `get_setting_from_config` has `-> Option<String>` and i would expect that function to always return `Option<String>` which is no longer the case due to `LIBREDDIT_PUSHSHIFT_FRONTEND` being different.

If i may ask a question: How is `LIBREDDIT_BANNER` different from `LIBREDDIT_PUSHSHIFT_FRONTEND`? It is even [defined to be string in the README](https://github.com/libreddit/libreddit#instance-settings).

Reply:
Banner is optional - if there's none set, it won't be displayed. It's listed as string, because not everyone reading the readme would know what `Option<String>` means

Pushshift frontend needs to be defined - it's essentially defined by default with a String rather than an Option<String> to let it fit better in the template

Reply:
@sigaloid Looks like i can't help here much even if i would love to. But beside Libreddit, i've never had a look at Rust code. Would love to have this in Libreddit but it seems stuck for now. Anyways, thanks allot for explaining and your work on this PR and Libreddit.

Reply:
@sigaloid Would it be an option to split this PR and try one for the default subscriptions only? It would be great to have this feature back in.